File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    row = [max(i) for i in sizes];
    column = [min(i) for i in sizes];
    
    # for r, c in sizes:
    #     if r > c:
    #         row.append(r);
    #         column.append(c);
    #     else:
    #         row.append(c);
    #         column.append(r);
    
    return max(row) * max(column)

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    const row = [];
    const column = [];
    // sizes.forEach(([r, c]) => {
    //     if(r > c){
    //         row.push(r)
    //         column.push(c)
    //     }
    //     else{
    //         row.push(c)
    //         column.push(r)
    //     }
    // })
    
    sizes.forEach(el => {
        row.push(Math.max(...el));
        column.push(Math.min(...el));
    })
    
    return Math.max(...row) * Math.max(...column)
}